### ISSUE TITLE
Ambiguous sentence in documentation

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5407]
|<img alt="LucianU's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| LucianU reported|
|-|-|
|Trac ID|trac#5407|
|Type|defect|
|Created|2011-11-23 14:42:48Z|

The note found at the [Writing Clients How To](http://twistedmatrix.com/documents/current/core/howto/clients.html#auto2/) is ambiguous. It looks like the last sentence got cut off.

Attachments:

* [clients.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/5ef/5ef23156c0aa1c931616c744bae90f8a2c00b7e7/520c9afa9580753ac20b3490ea31c16735f3583b.patch) (671 bytes) - added by exarkun on 2011-11-27 14:10:50Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5407 5407
type__defect defect
reporter__LucianU LucianU
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1322059368000000 1322059368000000
changetime__1322943786000000 1322943786000000
version__None None
owner__exarkun exarkun
cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

Thanks for catching this.  Fix attached, please review.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_3"></a>|@jesstess set owner to @exarkun|
|-|-|

Thanks for the patch, exarkun. Looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [33275]) Finish incomplete sentence about the interaction between endpoints and client factory methods.

Author: exarkun
Reviewer: jesstess
Fixes: [#5407](https://github.com/twisted/twisted/issues/5407)
